Client/play/static/player.app.js

```diff
@@ -175,18 +175,52 @@ var Status = React.createClass({displayName: "Status",
     return {
       playerAtLocation: false,
       huntComplete: false,
-      distanceToNextPin: 0.00
-    };
-  },  
+      distanceToNextPin: 0.00,
+      currentPin: null
+    }
+  },
+
+  getHuntStatus: function() {
+    var currentPin = Number.parseInt(this.props.hunt.get('currentPin'));   
```

Comment:
This will return undefined and does not mutate the state. It appears that overall this function does nothing. Is that intended.